spec/integration/connection_spec.rb

```diff
@@ -0,0 +1,29 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2018 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require_relative '../helpers/connection_helper'
+
+describe GoodData do
+  let(:login) { 'rubydev+admin@gooddata.com' }
+  before do
+    key = ENV['GD_SPEC_PASSWORD'] || ENV['BIA_ENCRYPTION_KEY']
+    File.write('dev-gooddata-sso.pub', GoodData::Helpers.decrypt(File.read('dev-gooddata-sso.pub.encrypted'), key))
+    File.write('rubydev_public.gpg', GoodData::Helpers.decrypt(File.read('rubydev_public.gpg.encrypted'), key))
+    File.write('rubydev_secret_keys.gpg', GoodData::Helpers.decrypt(File.read('rubydev_secret_keys.gpg.encrypted'), key))
+
+    `gpg --import dev-gooddata-sso.pub rubydev_public.gpg rubydev_secret_keys.gpg`
+  end
+
+  it 'can use SSO' do
+    rest_client = GoodData.connect_sso(login, 'test-ruby', GoodData::Environment::ConnectionHelper::DEFAULT_SERVER, {
+      server: GoodData::Environment::ConnectionHelper::DEFAULT_SERVER,
```

Comment:
Style/IndentHash: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.